### ISSUE TITLE
Suggestion Hard reset button somewhere in menus

### ISSUE THREAD

**Issue by Paralusion** [THEREALWWEFAN231/Inertia#767]
**Minecraft Version(s)**
All
**What is the suggestion**
Hard reset button that removes the current versions Inertia folder, not inside of the GUI but located on some menu, possibly in Inertia Options (possible with a option to back up the Inertia folder instead of straight up removing it)
**Why would this feature be useful**
If someone needs support and a hard reset is needed, this is way easier then telling them to navigate to the .minecraft folder, seek out the Inertia folder and tell them to remove it.

**Comment by MLGgamingpro** [comment]
I agree this would be useful for support, maybe adding a "backup" folder in `.minecraft/Inertia`.
